tests/Aspire.MySqlConnector.Tests/AspireMySqlConnectorExtensionsTests.cs

```diff
@@ -37,7 +44,9 @@ public void ReadsFromConnectionStringsCorrectly(bool useKeyed)
             host.Services.GetRequiredKeyedService<MySqlDataSource>("mysql") :
             host.Services.GetRequiredService<MySqlDataSource>();
 
-        Assert.Equal(ConnectionString, dataSource.ConnectionString);
+        string actualConnectionString = dataSource.ConnectionString;
+        string expectedConnectionString = new MySqlConnectionStringBuilder(ConnectionString).ConnectionString;
```

Comment:
`new MySqlConnectionStringBuilder(ConnectionString).ConnectionString` is repeated a few times in this file; it might be worth creating a `private string NormalizedConnectionString => new MySqlConnectionStringBuilder(ConnectionString).ConnectionString` property to encapsulate it?